### PR TITLE
Add task

### PR DESCRIPTION
**Pull request by IvanRamyk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,50 @@
-# Python boilerplate for GitHub tasks
+# Work with files 
 
-- Read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md) before start
```
Comment (MasterpieceElbow) [inline README.md:3]: Why did you delete guideline?

**Review by MasterpieceElbow — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (IvanRamyk) [reply README.md:3]: Fair enough

Comment (IvanRamyk) [reply README.md:3]: Fixed
